dnf-plugin-system-upgrade.spec

```diff
@@ -1,6 +1,9 @@
+%{!?dnf_lowest_compatible: %global dnf_lowest_compatible 1.1.2}
+%{!?dnf_not_compatible: %global dnf_not_compatible 2.0}
```

Comment:
Honestly, I'm not sure. All the other DNF related specs have it this way. I'm assuming that eventually there will be some kind of dnf macro thing or something like that.